src/ansys/motorcad/core/geometry.py

```diff
@@ -1164,19 +1164,7 @@ def length(self):
         float
             Length of arc
         """
-        radius, angle_1 = xy_to_rt(self.start.x, self.start.y)
-        radius, angle_2 = xy_to_rt(self.end.x, self.end.y)
-
-        if self.radius == 0:
-            arc_angle = 0
-        elif ((self.radius > 0) and (angle_1 > angle_2)) or (
-            (self.radius < 0) and angle_2 < angle_1
-        ):
-            arc_angle = angle_2 - (angle_1 - 360)
-        else:
-            arc_angle = angle_2 - angle_1
-
-        return self.radius * radians(arc_angle)
+        return abs(self.radius * radians(self.total_angle))
```

Comment:
This is in another PR too? 
https://github.com/ansys/pymotorcad/pull/338
We shouldn't commit this in this PR. Either remove manually or wait until #338 is in main and then merge and resolve conflicts

Reply:
Yes I think I found this issue while fixing the other one. I think best to wait until 338 is in main then merge this one